build/utils/chargeRestCaller.js

```diff
@@ -189,4 +189,27 @@ const createPaymentAsync = async request => {
   return result;
 };
 
-export { getChargeAsync, createChargeAsync, getRefundAsync, createRefundAsync, getCustomerAsync, createCustomerAsync, getTransactionAsync, createPaymentAsync };
\ No newline at end of file
+const confirmPaymentAsync = async request => {
```

Comment:
I believe you can avoid the word `Async` in the function name since the type will be inferred